api/judicialBooking/models/booking.model.ts

```diff
@@ -0,0 +1,15 @@
+export interface Booking {
+    userId: string;
+    appointmentId: string;
+    roleId: string;
+    contract_type_id: string;
```

Comment:
Any reason why we have moved to snake case or is this how the data will be returned from the endpoint?

Reply:
this is how data will return from api.